.kokoro/release.sh

```diff
@@ -16,6 +16,11 @@
 
 set -eo pipefail
 
+# Enable the publish build reporter
+python3 -m pip install gcp-releasetool
+python3 -m releasetool publish-reporter-script > /tmp/publisher-script; source /tmp/publisher-script
```

Comment:
I tried all manner of cleverness here, including:

```
python3 -m releasetool publish-reporter-script | source
```

and

```
source <(python3 -m releasetool publish-reporter-script)
```

Nothing seemed to work except this relatively inelegant solution.

Oh well, it's still just one line (kinda).